spec/models/option_set_import_spec.rb

```diff
@@ -29,6 +28,42 @@
     end
   end
 
+  context "with simple translations" do
+    let(:filename) { "translations_simple.csv" }
+
+    it "should be able to import" do
+      expect(import).to be_succeeded
+      option_set = import.option_set
+      expect(option_set.options[0].name).to eq("student")
+      expect(option_set.options[0].value).to eq(0)
+      expect(option_set.options[2].name).to eq("teacher")
+      expect(option_set.options[2].value).to eq(2)
+      expect(option_set.options[2].name_translations["en"]).to eq("teacher")
+      expect(option_set.options[2].name_translations["fr"]).to eq("professeure")
+      expect(option_set.options[2].name_translations["ar_EG"]).to eq("pwofesè")
+      expect(option_set.options[2].name_translations["foo"]).to eq(nil)
+      expect(option_set.options).to have(3).items
+    end
+  end
+
+  context "with mistaken translations" do
```

Comment:
Thanks for adding this. I actually had a failure one too, but it seemed to just gracefully handle things to my surprise so I removed it. Shouldn't have done that, so thanks for adding one in!